building-and-packaging/material/packaging_debian_and_spack_exercise.md

```diff
@@ -0,0 +1,99 @@
+# Packages with CPack and Spack
+
+This exercise is about packaging code with CPack and Spack. In this exercise we use adapted versions of the code from the previous week's exercise and create packages for it. The exercise is split into two parts: 1. CPack and 2. Spack. There are different GitHub repositories for the two subtasks.
+
+At the end of the exercise you find a section with hints and remarks. Make sure to check this section.
+
+Deadline: **Thursday, December 7th, 2021, 9:00**
+
+## 1. CPack
+
+**Attention/TODO**: Use [test repository](https://github.com/Simulation-Software-Engineering/cpack-exercise-test) for testing the exercise.
+
+The ultimate goals is to have a CMake/CPack configuration that allows us to generate packages for different targets. In this exercise we want to create a `tar.gz` package and a Debian `deb` package. We want to package this [HelloWorld example from GitHub](https://github.com/Simulation-Software-Engineering/cpack-exercise).
```

Comment:
Is "target" here used the correct way. Do you mean `cpackexample` and `cpackexamplelib`, or `TGZ` and `DEB`?

Reply:
I rephrase this. In my head it was "CPack target packages types" or so. It refers to us being able to build `tar.gz` and `deb` packages.